agentops/worker.py

```diff
@@ -47,6 +48,7 @@ def __init__(self, config: Configuration) -> None:
         self.thread = threading.Thread(target=self.run)
         self.thread.daemon = True
         self.thread.start()
+        self._session: Session | None = None
```

Comment:
Are there considerations for pulling in just the session_id or the whole thing? Not a big deal either way.